backend/private-graph/graph/schema.resolvers.go

```diff
@@ -6481,6 +6481,29 @@ func (r *queryResolver) WorkspaceInviteLinks(ctx context.Context, workspaceID in
 	return workspaceInviteLink, nil
 }
 
+// WorkspacePendingInvites is the resolver for the workspace_pending_invites field.
+func (r *queryResolver) WorkspacePendingInvites(ctx context.Context, workspaceID int) ([]*model.WorkspaceInviteLink, error) {
+	_, err := r.isAdminInWorkspace(ctx, workspaceID)
+	if err != nil {
+		return nil, nil
+	}
+
+	// TODO(spenny): filter to invites that are pending for users with emails
```

Comment:
I haven't been able to test this locally since I am always logged in as a demo@example.com

I need to check the following:
- are invites cleaned up when a user activates, or do we have a field to notify that a link is no longer active
- ensure that expired invites are not displayed

Reply:
Invites are deleted after a user activates, and AFAIK invites with email attendees do not have expiration dates
<img width="1186" alt="Screenshot 2023-05-26 at 6 46 53 PM" src="https://github.com/highlight/highlight/assets/17744174/efc57226-6730-48b6-8d2d-30bef577e2c7">